lib/fast_stats/means.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# A class for managing the arithmetic and geometric means
+# of positive numbers
+module FastStats
+  class Means
+
+    attr_reader :means
+
+    def initialize
+      @means = {}
+    end
+
+    def add(name, val)
+      mean_for(name) << val
+    end
+
+    def summary(round: Mean::DEFAULT_ROUND)
```

Comment:
My understanding is that the purpose of this gem is to avoid data-clumps. However, this method (which it seems is the main entry-point) returns just that: a data clump.

Looking at how it is used in ActionSprout code, this makes sense. I'm thinking right now that it might not make sense to change, as this API does actually work fairly well for how this code is used. However, it still feels like a code smell.

Reply:
Would it make more sense if instead of a `summary` `Means` provided a `fetch` or `[]` method and then in our FB gem we make a `PublicPostMetrics` value object that gets created by plucking out the metrics from a `Means` instance that we want (and the other metrics that are not means being calculated)?